Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenuBar.cpp

```diff
@@ -41,36 +46,29 @@ namespace AzToolsFramework
         return menuIterator->second;
     }
 
-    QMenuBar* EditorMenuBar::GetMenuBar()
-    {
-        return m_menuBar;
-    }
-
-    const QMenuBar* EditorMenuBar::GetMenuBar() const
-    {
-        return m_menuBar;
-    }
-
     void EditorMenuBar::RefreshMenuBar()
     {
-        m_menuBar->clear();
+        if (!m_mainWindow)
+        {
+            return;
+        }
+
+        m_mainWindow->menuBar()->clear();
```

Comment:
question: Can `menuBar()` ever be null?

Reply:
Nope Qt guarantees this will exist for us :)

```
Returns the menu bar for the main window. This function creates and returns an empty menu bar if the menu bar does not exist.
```